src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,40 @@
+package duke;
+
+public class Deadline extends Task {
+
+  private String date;
+
+  public Deadline(String description, String date) {
+    this.description = description;
+    this.date = date;
+    this.done = false;
+  }
+
+  public Deadline(String description, String date, boolean done) {
+    this.description = description;
+    this.date = date;
+    this.done = done;
```

Comment:
```suggestion
    this.isDone = done;
```
Do you think using isDone will conform better to our styleguide?